package.json

```diff
@@ -38,7 +37,7 @@
   "homepage": "https://github.com/gcanti/io-ts",
   "dependencies": {},
   "peerDependencies": {
-    "fp-ts": "^2.0.0"
+    "fp-ts": "^2.8.0"
```

Comment:
Does this qualify as a breaking change?

Reply:
@mmkal yes it does

@osdiab if I'm not mistaken we can avoid such a breaking change by keeping the `import-path-rewrite` step

Reply:
Cool, redid it without removing `import-path-rewrite`! Thanks for the tip.